xbmc/cores/DllLoader/Win32DllLoader.cpp

```diff
@@ -324,6 +324,9 @@ void Win32DllLoader::OverrideImports(const std::string &dll)
 
 bool Win32DllLoader::NeedsHooking(const char *dllName)
 {
+  if (!StringUtils::EndsWithNoCase(dllName, "libdvdread.dll"))
```

Comment:
Ah yes true that is the only exception. It shouldnt be hard to change that to use the callbacks though.